content/profiles/template-please-copy.md

```diff
@@ -1,33 +1,24 @@
 ---
-title: "Your Name"
```

Comment:
Hi Marieke!
Thanks for your profile!
However, you have edited the template :-). Could you make a copy of this file and title it `mariekie-coetzee.md` and revert this one (`template-please-copy.md`) back?

Layla

Reply:
Hello Layla!

Nooo thats soo embarrasing 🙈  really sorry about that!

Reply:
Oh don't worry!! Happens to the best of us 🤗🤗

Reply:
I've merged your changes - but you haven't added your profile and cover image. May have disappeared in the reversion. Just make another PR to add them :-)

Reply:
o dear!  thanks for pointing it out!